web/src/components/PoC.js

```diff
@@ -1,21 +1,39 @@
 import PropTypes from 'prop-types';
 import React, { Component, Fragment } from 'react';
+import Icon from '@fortawesome/react-fontawesome';
 
 import Collapsible from './Collapsible';
-import TopNav from '../containers/TopNav';
+import FormActivityGoals from './FormActivityGoals';
+import { faHelp, faChevronDown, faSignOut } from './Icons';
+import { getParams, stateLookup } from '../util';
 
 const sections = [
   'Short Summary of Activity',
   'Activity Description',
   'Goals and Objectives'
 ];
 
-const Sidebar = ({ activities }) => {
+const activityDisplay = (a, i) => {
+  let display = `Activity ${i}`;
+  if (a.name) display += `: ${a.name}`;
+  return `${display} (${a.type})`;
+};
+
+const Sidebar = ({ activities, place }) => {
   const linkClass = 'inline-block white text-decoration-none truncate';
 
   return (
     <div className="site-sidebar bg-navy">
       <div className="p2 xs-hide sm-hide">
+        <div className="mb3 center">
+          <img
+            src={`/static/img/${place.id}.svg`}
+            alt={place.name}
```

Comment:
Minor quibble, but should this alt tag be empty since the image doesn't add anything to the content?  Or maybe it does add context...  Hmm...  🤔

Reply:
i think it does add to the content/context